style.css

```diff
@@ -0,0 +1,47 @@
+body {
+	background: #181e6b;
+	overflow: hidden;
+}
+.boxes {
+  height: 150px;
+  width: 150px;
+  /*border: 1px solid #fff;*/
```

Comment:
remember to remove your commented code when you submit for a "production" environment